public/pages/Overview/containers/AnomalyDetectionOverview.tsx

```diff
@@ -281,7 +281,7 @@ export function AnomalyDetectionOverview(props: AnomalyDetectionOverviewProps) {
         <EuiFlexGroup justifyContent="spaceBetween">
           <EuiFlexItem grow={false}>
             <EuiTitle size="l" data-test-subj="overviewTitle">
-              <h1>Anomaly detection</h1>
+              <h1>Get started</h1>
```

Comment:
Is this supposed to be under the flag?

Reply:
Perhaps it's fine to just change the text to `Get started` as it describes perfectly what the page is doing :)

Reply:
confirmed with UX designers, we need this change for 2.16 and it doesn't need to be under a flag